packages/database/test/database.test.ts

```diff
@@ -62,6 +62,15 @@ describe('Database Tests', () => {
     expect(db.ref().toString()).to.equal('https://foo.bar.com/');
   });
 
+  it('Can get database with multi-region URL', () => {
```

Comment:
Do you want to add a test with a non-lowercase namespace?

Reply:
Sounds like a good idea.